kyuubi-main/src/main/scala/org/apache/kyuubi/engine/ProcBuilder.scala

```diff
@@ -65,7 +65,7 @@ trait ProcBuilder {
   // Visible for test
   private[kyuubi] var logCaptureThread: Thread = _
 
-  private lazy val engineLog: File = ProcBuilder.synchronized {
+  private[kyuubi] lazy val engineLog: File = ProcBuilder.synchronized {
     val engineLogTimeout = conf.get(KyuubiConf.ENGINE_LOG_TIMEOUT)
     val currentTime = System.currentTimeMillis()
     val processLogPath = workingDir
```

Comment:
Scala support SAM since 2.12, we can simplify above code as
```
val totalExistsFile = processLogPath.toFile.listFiles { (_, name) => name.startsWith(module) }
```

Reply:
magic !